lib/bom.js

```diff
@@ -714,7 +717,30 @@ declare class AudioContext {
   decodeAudioData(arrayBuffer: ArrayBuffer, decodeSuccessCallback: Function, decodeErrorCallback: Function): void;
   decodeAudioData(arrayBuffer: ArrayBuffer): Promise<AudioBuffer>;
   resume(): Promise<void>;
```

Comment:
It looks like `resume` is actually defined on `AudioContext`

Reply:
ah I overlooked it. removed on ` [4e32402](/facebook/flow/pull/4570/commits/4e3240273a399ba2e72e0f7362299fb7ad00ba3b)`